generators/structure/index.js

```diff
@@ -0,0 +1,36 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    structure() {
+      const self = this;
+
+      this.log(yosay(
+        'Create Folder Structure'
+      ));
+      return this.prompt([{
```

Comment:
This is basic folder structure, but from future point of view we may need to add options for `mobile app structure` and ` web app structure`, So I have used a `checkbox` here.